### PR TITLE
Aggregate all lambda information to a single map

### PR DESCRIPTION
**Pull request by julienduchesne**

### CODE REVIEW

File: localstack/services/awslambda/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class LambdaFunction(object):
```
Comment (whummer) [inline localstack/services/awslambda/lambda_function.py]: Can we please extend the existing `LambdaFunction` class here: https://github.com/localstack/localstack/blob/master/localstack/utils/aws/aws_models.py#L159
We can rename the `id` parameter to `arn` in the constructor of that that class.

**Comment by whummer** [comment]
This is great @julienduchesne , thanks! I only added a minor comment, would be great if we could re-use the existing model class.

**Comment by julienduchesne** [comment]
Alright, I was not aware that there was an existing class.

**Comment by julienduchesne** [comment]
@whummer. Done

**Comment by whummer** [comment]
Thanks again!

**Pull request merged**